Library/Formula/gcviewer.rb

```diff
@@ -0,0 +1,27 @@
+class Gcviewer < Formula
+  homepage "https://github.com/chewiebug/GCViewer"
+  url "https://github.com/chewiebug/GCViewer/zipball/master"
```

Comment:
You're going to need a versioned tarball to get this into Homebrew.

`https://github.com/chewiebug/GCViewer/archive/1.34.1.tar.gz` is a good bet.